akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/directives/RespondWithDirectivesExamplesSpec.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.http.scaladsl.server
+package directives
+
+import akka.http.scaladsl.server.UnacceptedResponseContentTypeRejection
+import akka.http.scaladsl.model._
+import headers._
+
+class RespondWithDirectivesExamplesSpec extends RoutingSpec {
+
+  "respondWithHeader-examples" in {
+    val route =
+      path("foo") {
+        respondWithHeader(RawHeader("Funky-Muppet", "gonzo")) {
+          complete("beep")
+        }
+      }
+
+    Get("/foo") ~> route ~> check {
+      header("Funky-Muppet") shouldEqual Some(RawHeader("Funky-Muppet", "gonzo"))
+      responseAs[String] shouldEqual "beep"
+    }
+  }
+
+  "respondWithHeaders-examples" in {
+    val route =
+      path("foo") {
+        respondWithHeaders(RawHeader("Funky-Muppet", "gonzo"), Origin(Seq(HttpOrigin("http://spray.io")))) {
```

Comment:
I'll remove spray from everywhere.
The remaining references you still find in here are due to the yet (partially) unfinished migration of the directives docs.